src/oceanbox.cpp

```diff
@@ -275,7 +275,7 @@ void oceanbox::compute_fluxes( const unitval current_Ca, const double yf, const
             closs_total = closs_total + compute_connection_flux( i, yf );
         } // for i
 
-        if( 0 /* osc */ ) {
+        if( /* DISABLES CODE */ (0) /* osc */ ) {
```

Comment:
is all the code within this if statement disabled? when would we want to / need to enable it?

Reply:
Yes. It was debugging code used by Corinne. We could remove entirely, but might want for future issues?

Reply:
Wait so this is still part of the ocean component code that is used (cause there is some ocean stuff that is floating around not used or is this is) but the code in the `if statement` is helpful when debugging?

Reply:
Yes, the `if` block is helpful (was helpful, anyway) when debugging instabilities in the ocean boxes.

Reply:
ahhh I thought that it was legacy code from back in the day pre doeclim ocean. 

If it is helpful code to use in debugging it would probably be helpful to have a comment stating that